framework/widget/src/main/java/org/apache/ofbiz/widget/model/ModelFormField.java

```diff
@@ -2799,6 +2811,26 @@ public String getValue(Map<String, Object> context) {
             return getModelFormField().getEntry(context);
         }
 
+        /**
+         * Gets disabled.
+         * @return the disabled
+         */
+        public FlexibleStringExpander getDisabled() {
+            return disabled;
+        }
+
+        /**
+         *
+         * @param context the context
+         * @return evaluated value
+         */
+        public String getDisabled(Map<String, Object> context) {
```

Comment:
Should return a boolean based on whether the string evaluates to 'true', 'false', or empty (which would also be false).